qkernel/src/kernel_def.rs

```diff
@@ -230,7 +231,25 @@ pub fn HyperCall64(type_: u16, para1: u64, para2: u64, para3: u64, para4: u64) {
 #[cfg(target_arch = "aarch64")]
 #[inline(always)]
 pub fn HyperCall64(type_: u16, para1: u64, para2: u64, para3: u64, para4: u64) {
-    // TODO add HyperCall64
+    // use MMIO to cause VM exit hence "hypercall"
```

Comment:
Punctuation is off (alternative [patch](https://paste.sr.ht/blob/f83198f31e3f4cc30cee816e723b8d6e8e2e36d1)).